src/pb.rs

```diff
@@ -233,6 +237,21 @@ impl<T: Write> ProgressBar<T> {
         self.width = w;
     }
 
+    /// Set max refresh rate, above which the progress bar will not redraw, or `None` for none.
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
+    /// let mut pb = ProgressBar::new(...);
+    /// pb.set_max_refresh_rate(Some(100));
+    /// ```
+    pub fn set_max_refresh_rate(&mut self, w: Option<Duration>) {
```

Comment:
That would not describe what this function does, because `set_refresh_rate()` implies that it will refresh at *exactly* the given rate, `set_max_refresh_rate()` means that the refresh rate might be smaller (i.e. longer time between `draw()` calls)